src/Microsoft.Tye.Extensions/Dapr/DaprExtension.cs

```diff
@@ -271,23 +271,23 @@ private string GetDaprExecutablePath()
             // Starting with dapr version 11, dapr is installed in user profile/home.
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                var windowsPath = Environment.ExpandEnvironmentVariables("%USERPROFILE%/.dapr/bin/daprd.exe");
+                var windowsPath = Environment.ExpandEnvironmentVariables("%HOMEDRIVE%/dapr/dapr.exe");
```

Comment:
@rynowak is there any plan to move dapr.exe outside of the %HOMEDRIVE%?

Reply:
I would think not since it's 1.0.